tests/album-create.test.js

```diff
@@ -0,0 +1,57 @@
+// tests/album-create
+
+const { expect } = require('chai');
+const request = require('supertest');
+const getDb = require('../src/services/db');
+const app = require('../src/app');
+
+describe('create album', () => {
+  let db;
+  let artistEntryId;
+
+  beforeEach(async () => {
+    db = await getDb();
+    //create artist - required for album creation
+    await db.query('INSERT INTO Artist (name, genre) VALUES(?,?)', [
+        'Dire Straits',
+        'rock',
+    ]);
+    
+    // retrieve artist.Id from db for 'Dire Straits' 
+    // and assign id to artistRefId
+    const [[artistEntries]] = await db.query(
+      `SELECT * FROM Artist WHERE name = 'Dire Straits'`
+    );
+    artistRefId = artistEntries.id;
+  });
+
+  afterEach(async () => {
+    await db.query('DELETE FROM Album');
+    await db.end();
+  });
+
+  describe(`/artist/:artistId/album`, () => {
+    describe('POST', () => {
+      it('creates a new album in the database', async () => {
```

Comment:
Could a test be added here to assert the failure state if the artist id in the request doesnt exist in the database?